src/components/Sheet/SheetContent.tsx

```diff
@@ -140,14 +140,20 @@ class SheetContent extends React.Component<SheetContentInnerProps, SheetContentS
 
         return (
             <React.Fragment>
-                {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
                 <div
                     ref={this.veilRef}
                     className={sheetBlock('veil', veilTransitionMod)}
                     onClick={isAnimating ? undefined : this.onVeilClick}
                     onTransitionEnd={this.onVeilTransitionEnd}
+                    role="presentation"
```

Comment:
There is no need to specify this. div has no semantics, and hasn't contain any children

Reply:
Thank you for information! 
But I don't know how to avoid eslint-disable-next-line jsx-a11y/click-events-have-key-events because this div must be clickable. Maybe I could rewrite the component as a functional and use the useLayout instead of the usual handler? I also think it would be nice to add FocusTrap to the sheet content

Reply:
Oh I see, then let's keep it